packages/purpose-template-process/src/model/errors.ts

```diff
@@ -0,0 +1,12 @@
+import { makeApiProblemBuilder } from "pagopa-interop-models";
+
+export const errorCodes = {
+  purposeTemplateNotFound: "0001",
+};
+
+export type ErrorCodes = keyof typeof errorCodes;
+
+export const makeApiProblem = makeApiProblemBuilder(errorCodes, {
+  problemErrorsPassthrough: true,
+  forceGenericProblemOn500: true,
+});
```

Comment:
No need to set these:
- Hiding all unmapped errors with `forceGenericProblemOn500` is a feature we enable only in frontier services (BFF / M2M gw)
- `problemErrorsPassthrough` is true by default

```suggestion
export const makeApiProblem = makeApiProblemBuilder(errorCodes);
```